packages/block-library/src/latest-posts/index.php

```diff
@@ -21,6 +43,9 @@ function render_block_core_latest_posts( $attributes ) {
 		'suppress_filters' => false,
 	);
 
+	$block_core_latest_posts_excerpt = $attributes['excerptLength'];
+	add_filter( 'excerpt_length', 'get_block_core_latest_posts_excerpt', 999 );
```

Comment:
Should we want to fall back to the default behavior of excerpt length when the attribute isn't explicitly assigned?

Reply:
hmm, indeed, this might not be always set.

Reply:
I've tested without setting it seems that LP block always has that attribute at render time.